packages/default-theme/cms/blocks/CmsBlockDefault.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <CmsGenericElement
+    v-if="getContent"
+    :content="getContent"
+    class="cms-block-default"
+  />
+</template>
+
+<script>
+import CmsGenericElement from 'sw-cms/CmsGenericElement'
+
+export default {
+  components: {
+    CmsGenericElement,
+  },
+  name: 'CmsBlockDefault',
+  props: {
+    content: {
+      type: Object,
+      default: () => ({}),
+    },
+  },
+  computed: {
+    getSlots() {
+      return this.content.slots || []
+    },
+    getContent() {
+      return this.getSlots.length && this.getSlots[0]
+    },
+  },
+}
```

Comment:
this logic is exactly the same as we have in `packages/default-theme/cms/blocks/CmsBlockCategoryNavigation.vue` 
Can it be provided as a mixin or something else to avoid redundant code? ;) Or I don't get it :D

Reply:
good point, created issue for that :) https://github.com/DivanteLtd/shopware-pwa/issues/636